main.tf

```diff
@@ -74,16 +145,22 @@ module "obs_testnet_vpc" {
   deploy_ec2_instance_db                 = false
   deploy_rds_db                          = true
   xchain_settings = {
-    enabled      = true
-    docker_image = local.xchain_indexer_testnet_docker_image
-    config       = "testnet"
-    omni_config = {
-      omni_rpc = local.omni_testnet_rpc
+    enabled             = true
+    docker_image        = local.xchain_indexer_testnet_docker_image
+    config_file_name    = "testnet"
```

Comment:
kept for backward compatibility, once approach is tested on staging we can dithc config_file_name completely